### ISSUE TITLE
Suggest arrays when running a string containing spaces

### ISSUE THREAD

**Issue by l0b0** [koalaman/shellcheck#2591]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/usr/bin/env bash
path='foo bar'
command='find -maxdepth 1'
command="${command} ${path}"
$command
```

#### Here's what shellcheck currently says:

> No issues detected!

#### Here's what I wanted or expected to see:

> Use an array to construct a command.

[Rationale](https://paperless.blog/bash-lints-beyond-shellcheck): See [I’m trying to put a command in a variable, but the complex cases always fail!](https://mywiki.wooledge.org/BashFAQ/050).

Exceptions: [Passing arguments to SSH is even more complex](https://www.chiark.greenend.org.uk/~cjwatson/blog/ssh-quoting.html); not sure how easy it would be to detect SSH usage.
